browser/static/js/main_zstack.js

```diff
@@ -1304,7 +1304,13 @@ function action(action, info, frame = current_frame) {
         seg_array = payload.imgs.seg_arr;
 
         seg_image.src = payload.imgs.segmented;
```

Comment:
Won't this image be missing if channels have been changed? Seems like this should also be loaded only `if (payload.imgs.hasOwnProperty('segmented'))`. (Same thing in main_track.js)

Reply:
Makes sense. I've added the same conditional for segmented and seg_arr for both zstack.js and track.js.